analysis/active_analyses.R

```diff
@@ -71,6 +71,7 @@ cognitive_impairment <- c("out_date_any_dementia")
 
 parkinson_disease <- c("out_date_any_dementia")
```

Comment:
Please could you give the values `vascular_risk`, `parkinson_risk`, `cognitive_impairment` and `parkinson_disease` more informative names? Code such as `parkinson_disease <- c("out_date_any_dementia")` is potentially confusing.

Reply:
Added suffix "_sub_out" and a note indicating which covariate was removed.
out_date_cognitive_impairment_symptons cannot be renamed (preprocess already ran). Rename should be made in Stata.